src/HIndent.hs

```diff
@@ -1,165 +1,161 @@
 {-# OPTIONS_GHC -fno-warn-name-shadowing #-}
 {-# LANGUAGE RankNTypes #-}
-{-# LANGUAGE OverloadedStrings, ScopedTypeVariables, PatternGuards #-}
+{-# LANGUAGE OverloadedStrings, ScopedTypeVariables, PatternGuards
+  #-}
 
 -- | Haskell indenter.
-
 module HIndent
-  (-- * Formatting functions.
-   reformat
-  ,prettyPrint
-  ,parseMode
-  -- * Testing
-  ,test
-  ,testFile
-  ,testAst
-  ,testFileAst
-  ,defaultExtensions
-  ,getExtensions
-  )
-  where
+  ( reformat
+  , prettyPrint
+  , parseMode
+  , test
+  , testFile
+  , testAst
+  , testFileAst
+  , defaultExtensions
+  , getExtensions
+  ) where
```

Comment:
Can we keep the comments for separation here?

Reply:
I removed the first comment because of https://github.com/mihaimaruseac/hindent/issues/600 and the rest because I thought it was somewhat strange to have headlines in the middle of a list and not have one at the first line. Should we first resolve #600?

Reply:
I think so, I'd prefer to keep the documentation blocks